sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -350,10 +351,39 @@ def test_enet_cv_positive_constraint():
     assert_true(min(enetcv_constrained.coef_) >= 0)
 
 
+def test_uniform_targets():
+    enet = ElasticNetCV(fit_intercept=True, n_alphas=3)
+    m_enet = MultiTaskElasticNetCV(fit_intercept=True, n_alphas=3)
+    lasso = LassoCV(fit_intercept=True, n_alphas=3)
+    m_lasso = MultiTaskLassoCV(fit_intercept=True, n_alphas=3)
+
+    models_single_task = (enet, lasso)
+    models_multi_task = (m_enet, m_lasso)
+
+    rng = np.random.RandomState(0)
+
+    X_train = rng.random_sample(size=(10, 3))
+    X_test = rng.random_sample(size=(10, 3))
+
+    y1 = np.empty(10)
+    y2 = np.empty((10, 2))
+
+    for model in models_single_task:
+        for y_values in (0, -5, 5, 4.5, -4.5):
+            y1.fill(y_values)
+            assert_array_equal(model.fit(X_train, y1).predict(X_test), y1)
+
+    for model in models_multi_task:
+        for y_values in (0, -5, 5, 4.5, -4.5):
+            y2[:, 0].fill(y_values)
+            y2[:, 1].fill(2 * y_values)
```

Comment:
Can you also check that the grid of alphas obtained are the same?
Also I think it is sufficient to just check for maybe two inputs for `y`, i.e 0 or 5.
After that you have my +1 for merge.